spec/status_spec.rb

```diff
@@ -1,51 +1,49 @@
-require 'spec_helper'
-
 describe CXML::Status do
-  describe 'instance' do
-    it { should respond_to :code }
-    it { should respond_to :text }
-    it { should respond_to :xml_lang }
-    it { should respond_to :success? }
-    it { should respond_to :failure? }
+  describe "instance" do
+    it { is_expected.to respond_to :code }
+    it { is_expected.to respond_to :text }
+    it { is_expected.to respond_to :xml_lang }
+    it { is_expected.to respond_to :success? }
+    it { is_expected.to respond_to :failure? }
   end
 
-  describe '#initialize' do
-    it 'assigns attributes from string' do
-      str = '<Status xml:lang="en-US" code="200" text="OK"></Status>'
+  describe "#initialize" do
+    it "assigns attributes from string" do
+      str = "<Status xml:lang='en-US' code='200' text='OK'></Status>"
       status = CXML::Status.new(str)
 
-      status.code.should eq(200)
-      status.xml_lang.should eq('en-US')
-      status.text.should eq('OK')
+      expect(status.code).to eq(200)
+      expect(status.xml_lang).to eq("en-US")
+      expect(status.text).to eq("OK")
     end
 
-    it 'assigns attributes from hash' do
-      hash = {'xml:lang' => 'en-US', 'code' => "200", 'text' => 'OK'}
+    it "assigns attributes from hash" do
+      hash = {"xml:lang" => "en-US", "code" => "200", "text" => "OK"}
       status = CXML::Status.new(hash)
 
-      status.code.should eq(200)
-      status.xml_lang.should eq('en-US')
-      status.text.should eq('OK')
+      expect(status.code).to eq(200)
+      expect(status.xml_lang).to eq("en-US")
+      expect(status.text).to eq("OK")
     end
   end
 
-  describe '#success?' do
-    it 'returns true on 2xx codes' do
-      CXML::Status.new('code' => '200').success?.should be_true
-      CXML::Status.new('code' => '201').success?.should be_true
-      CXML::Status.new('code' => '281').success?.should be_true
+  describe "#success?" do
+    it "returns true on 2xx codes" do
+      expect(CXML::Status.new("code" => "200").success?).to be_truthy
+      expect(CXML::Status.new("code" => "201").success?).to be_truthy
+      expect(CXML::Status.new("code" => "281").success?).to be_truthy
```

Comment:
It is preferred to avoid be_truthy matcher because it will accept everything expect false and nil.
So if ```status.success?```returns "potato" then this test would still pass.
Same for be_falsey, because it accepts nil.